app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class CommentsController < ApplicationController
+
+  def index
+    @comments = Comment.all
+  end
+
+  def new
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = Comment.create!(comment_params.merge(post: @post))
```

Comment:
Have never tried doing it this way. I'm partial to the shorter `@post.comments.create( comment_params )` (especially because it's very readable).